packages/nx-sonarqube/src/executors/scan/utils/utils.ts

```diff
@@ -86,7 +86,13 @@ export async function determinePaths(
 
   deps.workspaceLibraries
     .filter((project) =>
-      options.skipImplicitDeps ? project.type !== DependencyType.implicit : true
+      context.projectName === project.name || (options.skipPaths.length ? options.skipPaths.some(path => project.sourceRoot.includes(path)) : true)
```

Comment:
Can you explain what you mean exactly by `skipPaths` property here with an example?

Reply:
we can separate the shared component sonar activities, so in MicroFrontend the config can be,
```
        "skipPaths": ["libs/shared/"],
        "skipDependencyTypes": ["dynamic"],
```
In this case, each micro frontend can have its own sonar project.